modules/meb_api/app/controllers/meb_api/v0/education_benefits_controller.rb

```diff
@@ -26,10 +27,14 @@ def claim_status
       end
 
       def submit_claim
-        render json:
-               { data: {
-                 'status': 'received'
-               } }
+        response = submission_service.submit_claim(params)
+
+        # @NOTE: Need front end to send in_progress_form_id as well as claimant_id
+        render json: {
+          data: {
+            'status': response.status
```

Comment:
Do you need to define the serializer: here?

Reply:
For the time being this is just sending a 200 back so i wasn't sure if it was needed.  I can add one though if it's the convention.

Reply:
I could go either way but if you will need it in the future, you may want to explicitly define it or make some sort of comment for future use

Reply:
Sounds good, I added to the note to mention the serializer.